lib/VersionHistory/VersionHistoryPane/VersionHistoryPane.js

```diff
@@ -87,6 +93,7 @@ const VersionHistoryPane = ({
     currentVersion,
     dataLostLabel,
     intl,
+    hiddenLabelsMap,
```

Comment:
Why do we need to provide an object instead of a list of fields?

Reply:
for performance reasons, map is faster than a list.

Reply:
Yes, but why don't you do this routine on `getFieldLabels` level?

Reply:
You mean, pass a list of labels and convert to map inside the Componet/Function?

Reply:
but not labels, I mean fields

Reply:
I see, your point. Sure, I can improve it by doing so. Thank you for the suggestion

Reply:
updated